assets/_js/custom-viz.js

```diff
@@ -0,0 +1,5 @@
+import Boeing from '/boeing/boeing'
```

Comment:
There's no need for a `custom-viz.js`. Any custom JS should only be included on the page it is needed in. Doing it this way means that the next time we make a custom visual and import it's files into this file, we're going to be importing a bunch of extra JS (for the other visuals) that aren't necessary.